### ISSUE TITLE
Add language: [language]

### ISSUE THREAD

**Issue by bcfg897mjy-ship-it** [w3c/wai-wcag-em-report-tool#314]
* [x] Add language to list of languages (in [`src/locales/index.json`](https://github.com/w3c/wai-wcag-em-report-tool/blob/main/src/locales/index.json))
* [x] Add folder with UI translations (in [`src/locales/[your-language]`](https://github.com/w3c/wai-wcag-em-report-tool/blob/main/src/locales))
* [x] Add translation of WCAG in supported versions (in [`src/locales/[your-language]/WCAG.json`](https://github.com/w3c/wai-wcag-em-report-tool/blob/main/src/locales))

(Full documentation on [How to add a language](https://github.com/w3c/wai-wcag-em-report-tool/wiki/How-to-add-a-language))

**Comment by bcfg897mjy-ship-it** [comment]
javascript:void%2520function()%7Bdocument.querySelectorAll(%2522%2523wai-styles,%2523wai-info-box,.image-span%2522).forEach(a=%253E%7Ba.remove()%7D),document.querySelector(%2522body%2522).insertAdjacentHTML(%2522afterbegin%2522,%2522%253Cstyle%2520id='wai-styles'%253E.image-span,%2523failure,%2523success%2520%7Bcolor%3Ablack;font-weight%3Abold;font-size%3Asmall;font-family%3ANoto%2520Sans,Trebuchet%2520MS,Helvetica%2520Neue,Arial,sans-serif;background-color%3A%2523eed009;margin%3A0%25202px;padding%3A2px;speak%3Aliteral-punctuation%7D%2523success%7Bposition%3Aabsolute;width%3A0;height%3A0;clip%3Arect(0,0,0,0);%7D%2523wai-info-box%7Bz-index%3A1000;color%3Ablack;font-family%3ANoto%2520Sans,Trebuchet%2520MS,Helvetica%2520Neue,Arial,sans-serif;border%3Asolid%25201px%2520%2523ddd;background-color%3A%2523fff;box-shadow%3A0%25204px%25208px%25200%2520rgba(0,0,0,0.2),0%25206px%252020px%25200%2520rgba(0,0,0,0.19);%7D%2523wai-info-box%2520header%7Bfont-weight%3A700;background-color%3A%2523f2f2f2;color%3A%2523005a6a;padding%3A8px%252016px;%7D%2523wai-info-box%2520header%2520a%7Bfloat%3Aright;text-decoration%3Anone%7D%2523wai-info-box%2520div%7Bpadding%3A8px%252016px;%7D.wai-more-info%7Bposition%3Afixed;bottom%3A5em;right%3A5em%7D.wai-good%7Boutline%3A5px%2520solid%2520%2523005A6A;padding%3A2px%7D.wai-bad%7Boutline%3A5px%2520solid%2520%2523c0272d;padding%3A2px%7D.wai-note%7Boutline%3A2px%2520dashed%2520%2523eed009%7D%253C/style%253E%2522),document.querySelectorAll(%2522img,%2520%5Brole=img%5D%2522).forEach(function(a)%7Bvar%2520b=%2522%2522;if(a.hasAttribute(%2522role%2522)%2526%2526(b+=%2522Role=%2527%2522+a.getAttribute(%2522role%2522)+%2522%2527%253Cbr%253E%2522),a.hasAttribute(%2522aria-label%2522)%2526%2526(b+=%2522%E2%9D%93aria-label=%2527%2522+a.getAttribute(%2522aria-label%2522)+%2522%2527%253Cbr%253E%2522),a.hasAttribute(%2522aria-describedby%2522))%7Bb+=%2522Described%2520by%2520ID(s)=%2522;var%2520c=a.getAttribute(%2522aria-describedby%2522),d=c.split(%2522%2520%2522);for(i=0;i%253Cd.length;i++)%7Bvar%2520e=document.querySelector(%2522%5Bid=%2527%2522+d%5Bi%5D+%2522%2527%5D%2522);b+=%2522%253Ca%2520href=%2527%2523%2522+d%5Bi%5D+%2522%2527%253E%2522+d%5Bi%5D+%2522%253C/a%253E%2522,i%253Cd.length-1%2526%2526(b+=%2522,%2520%2522),e%2526%2526(e.classList.add(%2522wai-note%2522),e.insertAdjacentHTML(%2522afterbegin%2522,%2522%253Cspan%2520class=%2527image-span%2527%253Eid=%2527%2522+d%5Bi%5D+%2522%2527%253C/span%253E%2522))%7D%7Dif(a.hasAttribute(%2522aria-labelledby%2522))%7Bb+=%2522Labelled%2520by%2520ID(s)=%2522;var%2520f=a.getAttribute(%2522aria-labelledby%2522),g=f.split(%2522%2520%2522);for(i=0;i%253Cg.length;i++)%7Bvar%2520h=document.querySelector(%2522%5Bid=%2527%2522+g%5Bi%5D+%2522%2527%5D%2522);b+=%2522%253Ca%2520href=%2527%2523%2522+g%5Bi%5D+%2522%2527%253E%2522+g%5Bi%5D+%2522%253C/a%253E%2522,i%253Cg.length-1%2526%2526(b+=%2522,%2520%2522),h%2526%2526(h.classList.add(%2522wai-note%2522),h.insertAdjacentHTML(%2522afterbegin%2522,%2522%253Cspan%2520class=%2527image-span%2527%253Eid=%2527%2522+g%5Bi%5D+%2522%2527%253C/span%253E%2522))%7D%7Da.classList.add(%2522wai-good%2522),a.hasAttribute(%2522alt%2522)%253F%2522A%2522==a.parentNode.nodeName%253F%2522%2522==a.getAttribute(%2522alt%2522)%253Fb+=%2522%253Cspan%2520style=%2527border-bottom:2px%2520solid%2520%2523003366;%2527%253E%E2%9D%93Empty%2520link%2520alt%2520text.%2520Ok%253F%253C/span%253E%2522:b+=%2522%253Cspan%2520style=%2527border-bottom:2px%2520solid%2520%2523003366;%2527%253E%E2%9C%93%2520Link%2520alt%2520text=%2527%2522+a.getAttribute(%2522alt%2522)+%2522%2527.%2520Suitable%253F%253C/span%253E%2522:b+=%2522%E2%9C%93%2520Alt%2520text=%2527%2522+a.getAttribute(%2522alt%2522)+%2522%2527.%2520Suitable%253F%2522:%2522A%2522==a.parentNode.nodeName%253F!a.hasAttribute(%2522aria-label%2522)%2526%2526!a.hasAttribute(%2522aria-labelledby%2522)%2526%2526!a.hasAttribute(%2522aria-describedby%2522)%2526%2526!a.hasAttribute(%2522title%2522)%2526%2526(a.classList.replace(%2522wai-good%2522,%2522wai-bad%2522),b+=%2522%253Cspan%2520style=%2527border-bottom:2px%2520solid%2520%2523003366;%2527%253E%E2%9D%8C%2520Link%2520image%2520missing%2520alt%2520text%253C/span%253E%2522):!a.hasAttribute(%2522aria-label%2522)%2526%2526!a.hasAttribute(%2522aria-labelledby%2522)%2526%2526!a.hasAttribute(%2522aria-describedby%2522)%2526%2526!a.hasAttribute(%2522title%2522)%2526%2526(a.classList.replace(%2522wai-good%2522,%2522wai-bad%2522),b+=%2522%E2%9D%8C%2520Image%2520missing%2520alt%2520text%2522),a.hasAttribute(%2522title%2522)%2526%2526(b+=%2522%E2%9D%93Image%2520has%2520title=%2527%2522+a.getAttribute(%2522title%2522)+%2522%2527%2522),a.hasAttribute(%2522longdesc%2522)%2526%2526(b+=%2522%E2%9D%93Image%2520linked%2520to%2520a%2520long%2520description=%2527%2522+a.getAttribute(%2522longdesc%2522)+%2522%2527%2522),b%2526%2526a.insertAdjacentHTML(%2522afterend%2522,%2522%253Cspan%2520class=%2527image-span%2527%253E%2522+b+%2522%253C/span%253E%2522)%7D),document.querySelectorAll(%2522img,%2520%5Brole=img%5D%2522).length%253F(document.querySelector(%2522body%2522).insertAdjacentHTML(%2522beforeend%2522,%2522%253Cdiv%2520id=%2527success%2527%2520role=%2527alert%2527%253ESuccess!%2520Images%2520Found%2520on%2520Page:%2520%2522+document.title+%2522%253C/div%253E%2522),setTimeout(function()%7Bdocument.querySelector(%2522%2523success%2522).remove()%7D,3e3)):(document.querySelector(%2522body%2522).insertAdjacentHTML(%2522afterbegin%2522,%2522%253Cstrong%2520id=%2527failure%2527%2520role=%2527alert%2527%253ENo%2520Images%2520Found%2520on%2520Page:%2520%2522+document.title+%2522%253C/strong%253E%2522),setTimeout(function()%7Bdocument.querySelector(%2522%2523failure%2522).remove()%7D,6e3)),document.querySelector(%2522body%2522).insertAdjacentHTML(%2522beforeend%2522,%2522%253Caside%2520id=%2527wai-info-box%2527%2520class=%2527wai-more-info%2527%253E%253Cheader%253EFind%2520out%2520more%253Ca%2520href=javascript:document.querySelectorAll(%2527%2523wai-styles,%2523wai-info-box,.image-span%2527).forEach(function(el)%7Bel.remove()%7D);%2520aria-label=dismiss%253EX%253C/a%253E%253C/header%253E%253Cdiv%253E%253Ca%2520href=%2527https://w3.org/wai/test-evaluate/easy-checks/image-alt/%2527%253EChecking%2520Image%2520Alternative%2520Text%253C/a%253E%253C/div%253E%253C/aside%253E%2522)%7D();

**Comment by bcfg897mjy-ship-it** [comment]
https://github.com/w3c/wai-wcag-em-report-tool/issues/314
